composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewModel.kt

```diff
@@ -7,45 +7,57 @@ import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 
+data class Interval(
+    val exercises: List<String>,
+    val reps: Int
+)
```

Comment:
Very minor suggestion, we should make `Interval` its own file, like `Interval.kt`. We are going to use this in a lot of places and on other screens :)